### PR TITLE
Changelog updates

### PR DESCRIPTION
**Pull request by chalin**
Based on feedback from @alorenzen.

Staged at https://angulardart-org-dev.firebaseapp.com/changelog

### CODE REVIEW

File: src/changelog.md [hunk 1]
```diff
@@ -74,42 +73,25 @@ Also see:
...
-      final InjectorFactory injector = self.injector$Injector;
-      {% endprettify %}
-    - Used the app's root injector factory as an argument to `runApp()`:
-      <?code-excerpt "examples/ng/doc/toh-5/web/main.dart" retain="runApp" indent-by="0"?>
-      {% prettify %}
-      runApp(ng.AppComponentNgFactory, createInjector: injector);
-      {% endprettify %}
-
-    For example, see the `web/main.dart` file changes in the [toh-5 diff][].
+    For examples, see **Files Changed** in the [dependency-injection diff][].
```
Comment (kwalrath) [inline src/changelog.md]: Global: **Files Changed** -> **Files changed** (to match capitalization in GitHub's UI)

File: src/changelog.md [hunk 2]
```diff
@@ -74,42 +73,25 @@ Also see:
...
+
+  - Switched to static configuration of root injectors in `web/main.dart` files
```
Comment (kwalrath) [inline src/changelog.md]: @alorenzen I think this item/text is new. OK by you?

File: src/changelog.md [hunk 3]
```diff
@@ -74,42 +73,25 @@ Also see:
...
+    that formerly bootstrapped providers. For example, see `web/main.dart` under
+    the **Files changed** tab of the [toh-5 diff][].
 
 - Updated all `test/*_test.dart` files and their **Angular entry points**:
   - Dropped <del>`@Tags(const ['aot'])`</del>
-  - Dropped <del>`import 'package:angular/angular.dart'`<del>
-  - Added `import 'foo_test.template.dart' as ng;` in `foo_test.dart`
+  - Added `import 'foo.template.dart' as ng;` in the test file for `foo` components
```
Comment (alorenzen) [inline src/changelog.md]: I actually think we should stick with the `foo_test.template.dart` import, with the best practice of declaring a test component in your test file.

Comment (chalin) [reply src/changelog.md]: Done

**Comment by chalin** [comment]
Is this good to go now @alorenzen?

**Comment by chalin** [comment]
Travis is unhappy because of links broken into the API docs. Since this is unrelated, I'll merge this PR and address the API doc issue separately.

**Pull request merged**